### ISSUE TITLE
Entity xx implements device_state_attributes

### ISSUE THREAD

**Issue by gateriderz** [gadgetchnnel/entities_calendar#10]
**Describe the bug**
Warning in Home Assistant 2021.12
```
WARNING (MainThread) [homeassistant.helpers.entity] Entity <> implements device_state_attributes. Please report it to the custom component author.
```

**To Reproduce**
1. Install Home Assistant Core 2021.12
2. Check Logs

**Expected behavior**
No warning errors

**Log**
[homeassistant.helpers.entity] Entity ?? <> implements device_state_attributes. Please report it to the custom component author.

**HASS Environment**
HASS OS

**Additional context**
Change device_state_attributes --> extra_state_attributes

```
calendar.py
    @property
    def extra_state_attributes(self):
        """Return the device state attributes."""
        if self.data.event is None:
            # No tasks, we don't REALLY need to show anything.
            return None

        return {}
```

**Comment by gadgetchnnel** [comment]
I have just released version 0.0.8, which removes the **device_state_attributes** method (this wasn't acrually being used as no additional attributes were being set).

Can you please update to this and see if it works for you without that warning?

**Comment by gateriderz** [comment]
Updated to 0.0.8, no other warnings have been raised.
Thank you for your quick resolution.
